rpms/s3/s3rpm.spec

```diff
@@ -334,7 +334,6 @@ echo "[cortx-s3server-rpm] INFO: S3 RPM Clean section completed"
 /opt/seagate/cortx/s3/install/haproxy/haproxy_osver7.cfg
 /opt/seagate/cortx/s3/install/haproxy/haproxy_osver8.cfg
 /opt/seagate/cortx/s3/install/haproxy/logrotate/haproxy
-/opt/seagate/cortx/s3/install/haproxy/logrotate/haproxylogfilerollover.sh
```

Comment:
Remove this file from source base (git rm)

Reply:
done